dusty/cli/logs.py

```diff
@@ -15,4 +17,4 @@
 
 def main(argv):
     args = docopt(__doc__, argv)
-    return tail_container_logs(args['<service>'], args['--tail'])
+    return tail_container_logs(args['<service>'], args['-f'], args['--tail'])
```

Comment:
so i think @thieman moved away from having just normal tail be the default.  I think it is fine to change back, but might want to consult him